doc/sphinx/changes.rst

```diff
@@ -236,7 +236,7 @@ Ltac2 language
   fixes `#19590 <https://github.com/coq/coq/issues/19590>`_,
   by Pierre Roux).
 - **Fixed:**
-  `Ltac2 Eval` does not require to be focussed in a goal
+  `Ltac2 Eval` does not require to be focused in a goal
```

Comment:
```suggestion
  `Ltac2 Eval` no longer requires a focused goal
```

If that's accurate.  "Require to be focused" sounds awful..

Reply:
Maybe "require to be in a focused goal"? or simply "require a focused goal"?

Reply:
I will be on vacation, so please commit what sounds best and merge without waiting for me :wink:

Reply:
> Maybe "require to be in a focused goal"? or simply "require a focused goal"?

Ah, no.  The first is not grammatical, the second appears to negate the original text.